hack/make/validate-test

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+
+# Make sure we're not using gos' Testing package any more in integration-cli
+
+source "$(dirname "$BASH_SOURCE")/.validate"
+
+IFS=$'\n'
+files=( $(validate_diff --diff-filter=ACMR --name-only -- '*.go' | grep '^integration-cli/' || true) )
+unset IFS
+
+badFiles=()
+for f in "${files[@]}"; do
+	# we use "git show" here to validate that what's committed is formatted
```

Comment:
I think this comment was supposed to be above line 18, not here, and maybe a comment here as to _why_ `check_test.go` is a special case?